src/pages/otherTodo/OtherTodoPage.tsx

```
@@ -1,40 +1,95 @@
 
 const OtherTodoPage = () => {
   const navigate = useNavigate();
+  const { jobId } = useParams<{ jobId?: string }>();
+  if (!jobId || Number.isNaN(Number(jobId))) {
+    navigate(-1); // 또는 에러 페이지
+    return null;
+  }
+  const numericJobId = Number(jobId);
   const [currentPage, setCurrentPage] = useState(0);
 
+  const { data: jobDetail, isPending } = useJobDetailQuery(numericJobId);
```

ESLint

[error] React Hook "useJobDetailQuery" is called conditionally. React Hooks must be called in the exact same order in every component render. Did you accidentally call a React Hook after an early return?

(react-hooks/rules-of-hooks)